AsyncDisplayKit/Details/ASAbstractLayoutController.h

```diff
@@ -35,6 +35,8 @@ ASDISPLAYNODE_EXTERN_C_END
 
 - (NSSet *)indexPathsForScrolling:(ASScrollDirection)scrollDirection rangeMode:(ASLayoutRangeMode)rangeMode rangeType:(ASLayoutRangeType)rangeType __unavailable;
 
+- (void)allIndexPathsForScrolling:(ASScrollDirection)scrollDirection rangeMode:(ASLayoutRangeMode)rangeMode displaySet:(NSSet * _Nullable * _Nullable)displaySet preloadSet:(NSSet * _Nullable * _Nullable)preloadSet __unavailable;
```

Comment:
The compiler actually required I put in the _Nullable _Nullable or it would not assume I had any specifier there, even though there is a NONNULL assumption here.  I could clean this up somehow before landing.